declarations_site/spotter/utils.py

```diff
@@ -234,8 +239,15 @@ def send_found_notify(notify):
         'query': notify.task.query,
         'decl_list': load_declarations(notify.new_ids),
         'found_new': notify.found_new,
-        'site_url': settings.EMAIL_SITE_URL,
+        'site_url': settings.SITE_URL,
     }
+    if notify.email and notify.email.endswith('.chatbot'):
```

Comment:
Notification channel now in User.email, if use different from email notation we should create separate model or update base User model. I'm just trying to keep it simple.